server/start.go

```diff
@@ -277,6 +278,30 @@ func startStandAlone(svrCtx *Context, app types.Application, opts StartCmdOption
 
 	g, ctx := getCtx(svrCtx, false)
 
+	// Add the tx service to the gRPC router. We only need to register this
+	// service if gRPC is enabled.
+	if svrCfg.GRPC.Enable {
+		// create tendermint client
+		// assumes the rpc listen address is where tendermint has its rpc server
+		rpcclient, err := rpchttp.New(svrCtx.Config.RPC.ListenAddress, "/websocket")
```

Comment:
```suggestion
		rpcclient, err := rpchttp.New(clientCtx.NodeURI, "/websocket")
```

Shouldn't it be this instead?

Reply:
I would argue the correct thing is to use what is set by `simd start --rpc.laddr ...` which is svrCtx.Config.RPC.ListenAddress, but I might be missing context.
How is the clientCtx.NodeURI set?

Reply:
You are right! This is the server, I was looking at the client 👍🏾

Reply:
I think the --node flag makes a lot of sense here, since the semantics fit it well. I added it to the start command, I am happy with whatever of the two versions you prefer

Reply:
I disagree because the `--node` flag defaults to the value in the client.toml, which is very confusing.
I think your way made more sense `--with-comet=false ---rpc.laddr ...`
I do think it worth a quick mention here: https://github.com/cosmos/cosmos-sdk/blob/bff3104781cca3c409fecfa3efebd80a40eb48c3/docs/user/run-node/01-run-node.md#L146 so that user understand how to do it.

Reply:
Makes sense, the default value is a good argument

Reply:
Added a brief paragraph https://github.com/cosmos/cosmos-sdk/pull/18110/commits/9ed0aa7184d6d251e9f243a01e5253cf72ec1f32